apps/dashboard/src/utils/url.ts

```diff
@@ -0,0 +1,9 @@
+import { RedirectTargetEnum } from '@novu/shared';
+
+export const urlTargetTypes = [
+  RedirectTargetEnum.SELF,
+  RedirectTargetEnum.BLANK,
+  RedirectTargetEnum.PARENT,
+  RedirectTargetEnum.TOP,
+  RedirectTargetEnum.UNFENCED_TOP,
+];
```

Comment:
Can you also use this in the action picker since you added it?

Reply:
totally forgot to do that but it was the intention ;)